packages/components/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@terra-ui-packages/components",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "build": "tsc && tsc -p tsconfig.cjs.json",
```

Comment:
We build two different versions of each package (ES Modules and CommonJS).

Reply:
Should this have a comment inline describing why we need two versions?

Reply:
Off the top of my head, I'm not sure if NPM / Yarn can tolerate comments in package.json files. This could probably be clarified some by having the first `tsc` explicitly state the config file (`tsconfig.json`) vs relying on the default. The tsconfig files have comments about what they are for.